examples/data-objects/table-tree/src/Table/tableSchema.ts

```diff
@@ -97,21 +97,21 @@ export class ColumnProps extends schemaFactory.object("table-column-props", {
 }) {}
 export class Column extends TableSchema.column({
 	schemaFactory,
-	cell: Cell,
-	props: ColumnProps,
+	Cell,
```

Comment:
As noted in https://github.com/microsoft/FluidFramework/pull/24579#discussion_r2087833293 I think this idea (which was originally my idea) is a bad idea.